### ISSUE TITLE
Copilot doesn't work for remote files

### ISSUE THREAD

**Issue by igorkorneichuk** [SchoofsKelvin/vscode-sshfs#397]
Is there a workaround to make Copilot work with it?

**Comment by SchoofsKelvin** [comment]
No, the Copilot extension (or VS Code itself, depending on how they implemented it) would need to change their code, making use of VS Code's `FileSystemProvider` API that this extension uses. Similar to #42, many extensions expect a local real filesystem, usually to run executables on/against.
